definitions/linux.yaml

```diff
@@ -137,6 +137,116 @@ labels: [Authentication, Configuration Files]
 provides: [users.username]
 supported_os: [Linux]
 ---
+name: ProcMounts
+doc: Current mounted filesystems.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/mounts'
+labels: [System]
+supported_os: [Linux]
+---
+name: ProcSysFsSecurity
+doc: Filesystem hardening settings.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/fs/protected_hardlinks'
+      - '/proc/sys/fs/protected_symlinks'
```

Comment:
hardlinks and symlinks should be grouped together since they relate to the same thing.  the suid_dumpable should be split out.